compiler+runtime/src/cpp/clojure/core_native.cpp

```diff
@@ -276,6 +276,19 @@ namespace clojure::core_native
   {
     return __rt_ctx->eval(expr);
   }
+
+// TODO: implement opts
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
+  static object_ptr read_string(object_ptr const opts, object_ptr const str)
```

Comment:
I'd generally do something like `object_ptr const /* opts */`, since param names are optional in C++. You don't need to change this; just letting you know in case you hadn't seen this before.